multi/stm32l4-multi/flash.c

```diff
@@ -109,6 +109,17 @@ static inline int program_isValidAddress(uint32_t addr, size_t size)
 }
 
 
+static inline int otp_isValidAddress(uint32_t addr, size_t size)
+{
+	if ((addr >= OTP_ADDR) && (addr + size < (OTP_ADDR + OTP_SIZE)) &&
+		(addr % (2 * sizeof(uint32_t)) == 0) && size % (2 * sizeof(uint32_t)) == 0) {
```

Comment:
Our style is to break if's with 2 indentations. Sadly reviewdog will bark, but nevermind him
```suggestion
	if ((addr >= OTP_ADDR) && (addr + size < (OTP_ADDR + OTP_SIZE)) &&
			(addr % (2 * sizeof(uint32_t)) == 0) && size % (2 * sizeof(uint32_t)) == 0) {
```

Reply:
Done